day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -247,3 +247,35 @@ any external storage location.
 In the case of an all-in-one cluster, the etcd data directory is located in
 the `/var/lib/origin/openshift.local.etcd` directory.
 --
+* If etcd runs as a static pod, run the following commands:
++
+[IMPORTANT]
+====
+If you use static pods, use the v3 API.
+====
+.. Obtain the etcd endpoint IP address from the static pod manifest:
++
+----
+$ export ETCD_POD_MANIFEST="/etc/origin/node/pods/etcd.yaml"
+$ export ETCD_EP=$(grep https ${ETCD_POD_MANIFEST} | cut -d '/' -f3)
+----
+
+.. Obtain the etcd pod name:
++
+----
+$ oc login -u system:admin
+$ export ETCD_POD=$(oc get pods -n kube-system | grep -o -m 1 '\S*etcd\S*')
+----
+
+.. Take a snapshot of the etcd data in the pod and store it locally:
++
+----
+$ oc project kube-system
+$ oc exec ${ETCD_POD} -c etcd -- /bin/bash -c "ETCDCTL_API=3 etcdctl \
```

Comment:
By using oc, this means that this procedure only works when the API is up.

Reply:
@sdodson, is that an unfair assumption? @damemi, are there alternate commands that we can use, or do we need to make "the API server must be running" a prereq for backing up etcd? (If it needs to be a prereq for backing up, I'll need one of you to help me review the situation where we tell users to make an etcd backup and make sure that they're still valid.)

Reply:
I guess during normal operations we can assume API availability. But in disaster recovery scenarios it may not be for one reason or another. I'm interested in others feedback here.

Reply:
there is the option of doing a `docker ps | grep etcd`, finding the right pod, and doing `docker exec <pod>` instead of `oc exec`

Reply:
@damemi, I think if the ocp env is enabled CRIO, but not docker, then 'docker ps' command is not work also, is it a problem? Currently, I'm prefer to add prereq if API availability is mandatory.

Reply:
The master-exec scripts should handle all of that and currently deploy crio and docker specific implementations.

Reply:
I think the master-exec stript replacement for this would be: 

```
/usr/local/bin/master-exec etcd etcd ETCDCTL_API=3 etcdctl \ 
```

Instead of changing what were QA'ing we might simply add this as a note; or call out under where we show the snapshot command being run?

Reply:
I think that if @geliu2016 is willing, it would be better to document the one master-exec script method.